src/helper/csdsClient.ts

```diff
@@ -47,19 +49,19 @@ export class CsdsClient {
     }
 
     try {
-      const {baseURIs} = await request(this.getUrl(accountId), {
-        json: true,
+      const url = this.getUrl(accountId);
+      const {body} = await got<CSDSResponse>(url, {
+        responseType: 'json',
+        throwHttpErrors: true,
       });
-
+      const {baseURIs} = body;
```

Comment:
I believe this is undeeded you can set got to just resolve the body. There is no point in not doing that as we throw on HTTP errors anyways